sidebars.js

```diff
@@ -1087,97 +1087,80 @@ module.exports = {
-                "self-managed/setup/deploy/amazon/amazon-eks/eks-helm",
-                "self-managed/setup/deploy/amazon/amazon-eks/dual-region",
-                "self-managed/setup/deploy/amazon/amazon-eks/irsa",
+                {
+                  type: "category",
+                  label: "Amazon EKS",
+                  link: {
+                    type: "doc",
+                    id: "self-managed/setup/deploy/amazon/amazon-eks/amazon-eks",
+                  },
+                  items: [
+                    "self-managed/setup/deploy/amazon/amazon-eks/eks-terraform",
+                    "self-managed/setup/deploy/amazon/amazon-eks/eks-helm",
+                    "self-managed/setup/deploy/amazon/amazon-eks/dual-region",
+                    "self-managed/setup/deploy/amazon/amazon-eks/eks-eksctl",
+                    "self-managed/setup/deploy/amazon/amazon-eks/irsa",
+                  ],
+                },
+                {
+                  type: "category",
+                  label: "ROSA",
+                  link: {
+                    type: "doc",
+                    id: "self-managed/setup/deploy/amazon/openshift/terraform-setup",
+                  },
+                  items: [
+                    "self-managed/setup/deploy/amazon/openshift/terraform-setup",
+                    "self-managed/setup/deploy/amazon/openshift/terraform-setup-dual-region",
+                  ],
+                },
               ],
             },
             {
               type: "category",
-              label: "ROSA",
+              label: "Microsoft",
               link: {
                 type: "doc",
-                id: "self-managed/setup/deploy/amazon/openshift/terraform-setup",
+                id: "self-managed/setup/deploy/azure/microsoft-aks/microsoft-aks",
               },
               items: [
-                "self-managed/setup/deploy/amazon/openshift/terraform-setup",
-                "self-managed/setup/deploy/amazon/openshift/terraform-setup-dual-region",
+                "self-managed/setup/deploy/azure/microsoft-aks/aks-terraform",
+                "self-managed/setup/deploy/azure/microsoft-aks/aks-helm",
               ],
             },
             {
-              type: "category",
-              label: "Amazon Marketplace",
-              link: {
-                type: "doc",
-                id: "self-managed/setup/deploy/amazon/aws-marketplace",
-              },
-              items: [],
+              Google: ["self-managed/setup/deploy/gcp/google-gke"],
             },
-            "self-managed/setup/deploy/amazon/aws-ec2",
-          ],
-          "Microsoft (Azure)": [
             {
               type: "category",
-              label: "Microsoft AKS",
+              label: "Red Hat OpenShift",
               link: {
                 type: "doc",
                 id: "self-managed/setup/deploy/azure/microsoft-aks/microsoft-aks",
               },
```

Comment:
This will need adjustment

<img width="285" alt="image" src="https://github.com/user-attachments/assets/b84d3946-5cec-4be7-adf0-718047f895ea" />


What's happening now is that if I click on `Red Hat OpenShift` I open the `Microsoft` overview page.
Otherway around if I click on Microsoft I unravel the `Red Hat OpenShift` navbar.